public/stylesheet/pumpio.css

```diff
@@ -35,6 +35,10 @@ body {
   width: auto;
 }
 
+.navbar .btn-navbar span {
+    color: white;
```

Comment:
Two spaces, not four, for consistency with surrounding lines

Reply:
Sorry, I have emacs configuration by default in four XD, I will change